language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/contexts/BallerinaCompletionContextImpl.java

```diff
@@ -37,6 +40,8 @@
     private final List<Node> resolverChain = new ArrayList<>();
     private Token tokenAtCursor;
     private NonTerminalNode nodeAtCursor;
+    private boolean isContextTypeFound = false;
+    private Optional<TypeSymbol> contextType;
```

Comment:
Usually we don't use Optional as a field's type. Shall we change it?

Reply:
Done